src/components/Releases/ReleaseOverviewTab.tsx

```diff
@@ -90,31 +67,35 @@ const ReleaseOverviewTab: React.FC = () => {
             }}
           >
             <DescriptionListGroup>
-              <DescriptionListTerm>Labels</DescriptionListTerm>
+              <DescriptionListTerm>Created at</DescriptionListTerm>
               <DescriptionListDescription>
-                <MetadataList metadata={release.metadata?.labels} />
+                <Timestamp timestamp={release.metadata.creationTimestamp ?? '-'} />
               </DescriptionListDescription>
             </DescriptionListGroup>
             <DescriptionListGroup>
-              <DescriptionListTerm>Annotations</DescriptionListTerm>
-              <DescriptionListDescription>
-                <MetadataList metadata={release.metadata?.annotations} />
-              </DescriptionListDescription>
+              <DescriptionListTerm>Duration</DescriptionListTerm>
+              <DescriptionListDescription>{duration ?? '-'}</DescriptionListDescription>
             </DescriptionListGroup>
             <DescriptionListGroup>
-              <DescriptionListTerm>Created at</DescriptionListTerm>
+              <DescriptionListTerm>Release Plan</DescriptionListTerm>
+              <DescriptionListDescription>{release.spec.releasePlan}</DescriptionListDescription>
+            </DescriptionListGroup>
+            <DescriptionListGroup>
+              <DescriptionListTerm>Release Target (Managed Namespace)</DescriptionListTerm>
```

Comment:
quick question: I think not always the release target will be from a "managed namespace", right? For example, in this example I have a release in which it's from my own namespace:

<img width="1397" height="970" alt="Screenshot 2025-07-22 at 11 02 33" src="https://github.com/user-attachments/assets/4602959d-32d0-4c32-b039-8c765f28f529" />

I think it might be better to add a logic here to check if the release is from a managed namespace, and if so then we add this "(Managed Namespace)" text :)